geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/security/ProtobufSimpleAuthorizer.java

```diff
@@ -12,23 +12,34 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.internal.protocol.protobuf;
+package org.apache.geode.internal.protocol.protobuf.security;
 
+import org.apache.shiro.subject.Subject;
+import org.apache.shiro.util.ThreadState;
+
+import org.apache.geode.internal.security.SecurityService;
+import org.apache.geode.security.NotAuthorizedException;
 import org.apache.geode.security.ResourcePermission;
-import org.apache.geode.security.SecurityManager;
-import org.apache.geode.security.internal.server.Authorizer;
+import org.apache.geode.internal.protocol.protobuf.security.Authorizer;
 
 public class ProtobufSimpleAuthorizer implements Authorizer {
```

Comment:
This is Shiro specific. Please change the classname to reflect this.

Reply:
Same comment as with the authenticator.